### ISSUE TITLE
Who or what is your inspiration? 

### ISSUE THREAD

**Issue by Yuriysteam** [edenspiekermann/ama#6]
Hello, 

As a graphic design student I truly know how hard a creative process can be. That's why I have a question: what people or things do inspire you and how?

P.S. I bet one thing is a coffee machine ;)

**Comment by mattcav** [comment]
Users, I would like to say. Observing people and try to get their problems and needs; go deep, ask "why" and let the door open for unexpected insights. Meet real users, prepare yourself to listen. 

Sometimes this is pretty hard to integrate in a traditional design agency flow, but we're trying to move from it to a more user-centered design process. Run an user-centered project takes time, needs practice and a lot of efforts. You need to challenge your team, your clients and not least yourself. It's hard to manage and sometimes leads to questions you can't answer immediately.

In short, it's damn hard, but it's the the most and only effective method I know to design (digital?) products that work for real.

PS: coffee is overrated.

**Comment by HugoGiraudel** [comment]
> PS: coffee is overrated.

Says the Italian.

**Comment by christowski** [comment]
Hey Yuriy! Staying creative is hard sometimes, and for me, there are different stages and ways to keep the engine going. Generally I totally agree with Matteo: Talking to the users you are building a product for is crucial and insofar inspiring as they point you to problems or perspectives you might never have thought of before. It’s the best way to see things from a different angle.

If we are in a design phase where we can’t get direct user input, we of course do visual research from books and websites and, more importantly, talk to each other.That’s definitely the most valuable thing for me: one hour of wild brainstorming with other creatives is better than any inspirational pinterest board in the world. Sometimes it might even be as easy as leaving your desk, walking around the office and peek into other team’s processes and check what they are up to. We also have a weekly "Design Couch” where we discuss broader design topics and problems (while sitting on a couch).

And, finally, staying inspired also means: Doing other things than staring at screens. Berlin has a big culture of meetups and monthly events you can go to and meet people (e.g. Creative Mornings or up.front.ug). Hobbys are important as well: I definitely need a dedicated time off the screen and design world to see and do new things (read, sports, watch dull TV shows). Creativity is nothing you can turn on and off again from nine to five, but you can nurture it by keeping a journal, reflect things you see and do, draw, et cetera. Also: You don’t have to share everything you do. Sometimes, practice (like, redrawing a drawing you like, just to learn how it was made) is crucial, but it’s only for you. It helps you see things. I recommend the book “How to steal like an artist” by Austin Kleon. :) Have fun!
